tox.ini

```diff
@@ -42,12 +42,9 @@ commands = black --skip-string-normalization .
 [testenv:linters]
 skip_install = true
 deps =
-    flake8
-    flake8-colors
-    black
+    pre-commit
 commands =
-    black --skip-string-normalization --check .
-    flake8
+    pre-commit run -a
```

Comment:
Always prefer long syntax for option inside script. it's more explicit and cost nothing (ref: --all-files)
(and you make pre-commit on the same line than deps and commands)